server/middleware.go

```diff
@@ -455,6 +482,139 @@ func pagination(next http.Handler) http.Handler {
 	})
 }
 
+func logHttpRequest(log logger.Logger) func(next http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+			cfg, err := config.Get()
+			if err != nil {
+				return
+			}
+
+			ww := middleware.NewWrapResponseWriter(w, r.ProtoMajor)
+
+			if isLogHttpRequestEnabled(cfg.Logger) && cfg.Logger.Type != "" {
```

Comment:
I don't think we need to check if the Logger type is not nil here. The system should default to console logger where the user don't specify a different logger.